.gitlab-ci.yml

```diff
@@ -225,8 +225,7 @@ deploy-prod-canary:
   stage: deploy:canary
   extends:
     - .base-configuration
-    - .tags
-  when: manual
+    - .main
```

Comment:
could we avoid to deploy release PR merge?
it will be unnecessary noise

Reply:
Sure, the only solution I found is to check the commit title:
https://github.com/DataDog/browser-sdk/blob/d1a707e2a22dcd2a290fa8e87e064890d5321f18/.gitlab-ci.yml#L72